### PR TITLE
SYNKRON-264 Implement APIs in Iconic SDK

### PR DESCRIPTION
**Pull request by jinalpanchal123**
Implement following APIs:
- <https://api.sellercenter.com.br/docs/#/Orders/GET_v2-orders-order\\\>
- <https://api.sellercenter.com.br/docs/#/Orders/GET_v2-order-items-get\\\>
- <https://api.sellercenter.com.br/docs/#/Product/get_v2_products__productId__consignment_stocks\\\>
- <https://api.sellercenter.com.br/docs/#/Product/get_v2_product_shop_sku__shopSku_\\\>
- <https://api.sellercenter.com.br/docs/#/Product/get_v2_product_seller_skus\\\>
- <https://api.sellercenter.com.br/docs/#/Webhook/get_v2_webhook_entities\\\>

### CODE REVIEW

File: src/main/java/com/dft/theiconic/OrdersApi.java [hunk 1]
```diff
@@ -36,6 +35,19 @@ public OrdersResponse getOrders(HashMap<String, String> parameters) {
         return getRequestWrapped(request, handler);
     }
 
+    public OrdersResponse getOrders(String orderId) {
```
Comment (dftrakesh) [inline src/main/java/com/dft/theiconic/OrdersApi.java]: Rename method name getOrders to getOrderById

File: src/main/java/com/dft/theiconic/OrdersApi.java [hunk 2]
```diff
@@ -36,6 +35,19 @@ public OrdersResponse getOrders(HashMap<String, String> parameters) {
...
+        String path = String.format(V2_ORDERS, orderId);
+        HttpRequest request = get(path);
+        HttpResponse.BodyHandler<OrdersResponse> handler = new JsonBodyHandler<>(OrdersResponse.class);
+        return getRequestWrapped(request, handler);
+    }
+
+    public OrdersResponse getOrderItems() {
```
Comment (dftrakesh) [inline src/main/java/com/dft/theiconic/OrdersApi.java]: Create new class OrderItemsResponse

**Review by dftrakesh — CHANGES_REQUESTED** [review]

File: src/main/java/com/dft/theiconic/OrdersApi.java (hunk 2 above)
Comment (jinalpanchal123) [reply src/main/java/com/dft/theiconic/OrdersApi.java]: created sir

File: src/main/java/com/dft/theiconic/OrdersApi.java (hunk 1 above)
Comment (jinalpanchal123) [reply src/main/java/com/dft/theiconic/OrdersApi.java]: yes sir

**Pull request merged**